docs/migration/wsc53/php.md

```diff
@@ -80,3 +80,11 @@ If you implemented a custom search engine and relied on this method, you can inl
 You should take the time to verify the rewritten queries against the manual of the search engine to make sure it cannot generate malformed queries or security issues.
 
 See [WoltLab/WCF#3815](https://github.com/WoltLab/WCF/issues/3815) for details.
+
+## Styles
+
+The `StyleCompiler` is marked `final` now.
+It was noticed that the `$compiler` property was a design issue that leaked compiler state across multiple compiled styles, possibly causing misgenerated stylesheets.
+As the removal of the `$compiler` property effectively broke compatibility within the `StyleCompiler` and as the `StyleCompiler` never was meant to be extended it was marked final.
+
+See [WoltLab/WCF#3929](https://github.com/WoltLab/WCF/pull/3929) for details.
```

Comment:
```suggestion
Refer to [WoltLab/WCF#3929](https://github.com/WoltLab/WCF/pull/3929) for details.
```

Reply:
This one should be consistently adjusted everywhere.